crates/swc_ecma_ast/src/ident.rs

```diff
@@ -630,6 +632,48 @@ static RESERVED: phf::Set<&str> = phf_set!(
     "with",
 );
 
+static RESERVED_ATOMS: Lazy<FxHashSet<Atom>> = Lazy::new(|| {
```

Comment:
Can we define a macro instead?
One macro invocation per each group of strings.

```rust
define_set!(RESERVED, RESERVED_ATOMS, ["break", "case", "catch", ..., "with" ]);
```

I think we can keep maintainability good as well with this simple trick

Reply:
It can expand to a `phf_set` call and `static RESERVED_ATOMS: Lazy<FxHashSet<Atom>> = Lazy::new(|| {` call